src/List/list.store.js

```diff
@@ -95,11 +95,29 @@ export default Store.create({
         this.listSourceSubject.onNext(Observable.fromPromise(this.state.pager.getPreviousPage()));
     },
 
+    buildPager(listSearchPromiseWithPagination, listSearchPromiseWithAllUsers) {
+        return Promise.all([listSearchPromiseWithPagination, listSearchPromiseWithAllUsers]).then(
+            ([model, model1]) => {
+                model.pager.total = model1.valuesContainerMap.size;
```

Comment:
Unfortunately, this is not enough. The next/prev page handling may be also off. We seem to use `getNextPage`, `getPreviousPage`, we should override them (they are functions that return a promise, check the original implementation for reference)